src/app/pages/SearchResultsPage/SearchResultsView.tsx

```diff
@@ -31,12 +32,14 @@ export const SearchResultsView: FC<{
           wantedScope={wantedScope}
           searchTerm={searchTerm}
           searchResults={searchResults.filter(getFilterForLayer(wantedScope.layer))}
+          isPotentiallyIncomplete={isPotentiallyIncomplete}
           tokenPrices={tokenPrices}
         />
       ) : (
         <GlobalSearchResultsView
           searchTerm={searchTerm}
           searchResults={searchResults}
+          isPotentiallyIncomplete={isPotentiallyIncomplete}
```

Comment:
Does this get set if there are more results than MASS_LOAD_INDEXES

Reply:
Now it does.